panel/src/components/Misc/Progress.vue

```diff
@@ -41,7 +44,13 @@ export default {
 }
 
 .k-progress::-webkit-progress-value {
-  border-radius: 20px;
+  border-radius: inherit;
+  background: $color-focus;
+  transition: width 0.3s;
+}
+
```

Comment:
Let’s keep it DRY and combine both rule sets

Reply:
That doesn't work in this case since these are vendor specific selectors. In all cases one of them is not recognized, so the whole block is ignored

Reply:
https://stackoverflow.com/questions/50580986/why-cant-i-combine-pseudo-element-selectors/50581522#50581522

Reply:
I see. Good to know!